src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -14,6 +14,9 @@
     <Dependency Include="Microsoft.NETCore.Windows.ApiSets">
       <Version>$(WindowsAPISetPackageVersion)</Version>
     </Dependency>
+    <Dependency Include="runtime.native.System">
+      <Version>$(SystemNativePackageVersion)</Version>
+    </Dependency>
     <ProjectReference Include="..\Microsoft.NETCore.Jit\Microsoft.NETCore.Jit.pkgproj" />
```

Comment:
@ellismg, are you referring to including a runtime package? This is quite similar to how we do API sets on Windows.